src/components/services/Sftpgo.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <Generic :item="item">
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+        <template v-if="item.subtitle">
+          {{ item.subtitle }}
+        </template>
+        <template v-else>
+          <span v-if="versionstring">Version {{ versionstring }}</span>
+          <span v-if="activeConnections !== null && !this.showUpdateAvailable">
+            – Active connections: {{ activeConnections }}
```

Comment:
The idea here is to not show the line on small screen right? why `showUpdateAvailable` is named like that ? It's not related to updates ? or I'm missing something ? 
You may be able to reduce the message (for example, just `{{ activeConnections }} connections` or `{{ activeConnections }} clients`) to make it fit on most screen and avoid the need to have a matchMedia check.